File: C/1008/1008.c
#include <stdio.h>
int main (){
    int numero;
    double horas,salario, valor;
    
    scanf("%d",&numero);
    scanf("%lf",&horas);
    scanf("%lf",&valor);
    salario = valor * horas;
    printf("NUMBER = %d\n",numero);
    printf("SALARY = U$ %.2lf\n",salario);
    return 0;   
}

File: Php/1008.php
<?php
$num = (int)fgets(STDIN);
$horas = (int)fgets(STDIN);
$salario = (float)fgets(STDIN);
$receber = $horas * $salario;

echo "NUMBER = $num\n";
echo "SALARY = U$ " . number_format($receber,2,'.','') . "\n";
